### ISSUE TITLE
Customize loss function in a way that ignore the samples with large loss

### ISSUE THREAD

**Issue by vivichina** [fchollet/keras#5285]
I have a lot of sample images in three classes, but some of the labels of them are wrong, so the trained classifier can not achieve good performance, and correct the wrong labels needs to consume a lot of time.
To eliminate the effect of error labels, I intend to redefine the loss function like that: When calculating the losses of a batch of data, the largest loss is removed from the others at this batch. And then calculate the mean of the remaining elements as the loss value of this batch.
What should I do to achieve this goal, or is there any other better way to achieve the same purpose?
